pkgs/tools/networking/dsniff/default.nix

```diff
@@ -0,0 +1,82 @@
+{ stdenv, fetchFromGitLab, autoreconfHook, libpcap, db, glib, libnet, libnids, symlinkJoin, openssl_1_1 }:
+let
+  /*
+  dsniff's build system unconditionnaly wants static libraries and does not
+  support multi output derivations. We do some overriding to give it
+  satisfaction.
+  */
+  staticdb = symlinkJoin {
+    inherit (db) name;
+    paths = with db.overrideAttrs(old: { dontDisableStatic = true; }); [ out dev ];
+    postBuild = ''
+      rm $out/lib/*.so*
+    '';
+  };
+  pcap = symlinkJoin {
```

Comment:
Is it worth it to do all the symlink dance instead of just force-passing an extra `-I` and biting the relatively small closure bloat?

Reply:
The compiler finds libraries just fine, it is the configure script which does strange things by looking at the actual files instead of deferring finding libraries to the compiler.